src/main/java/com/maxmind/geoip2/record/AbstractRecord.java

```diff
@@ -0,0 +1,32 @@
+package com.maxmind.geoip2.record;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+
+public class AbstractRecord {
```

Comment:
maybe make this class abstract too? I think no instances will be created of this class, but rather its subclasses?